adapters/socketio_berkeley.c

```diff
@@ -650,7 +652,11 @@ int socketio_setoption(CONCRETE_IO_HANDLE socket_io, const char* optionName, con
         }
         else if (strcmp(optionName, "tcp_keepalive_time") == 0)
         {
+#ifdef __APPLE__
```

Comment:
UPDATE 2: I tried to use IPPROTO_TCP always per your comment @dcristoloveanu, but this file doesn't compile for ubuntu/gcc if I try to use `IPPROTO_TCP, TCP_KEEPALIVE`. So I'll keep the ifdef for now...